librubyfmt/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rubyfmt"
-version = "0.1.0"
+version = "0.6.7"
```

Comment:
should this `0.6.8.pre` maybe? the under development version is not version 0.6.7 (the pre string is how we did it in rspec)

Reply:
That isn't a valid semver version. I could do `-pre`, but setting the version on git to something different like this is pretty uncommon in the Rust world, since if someone *does* want to point at git for some reason this makes it more difficult for them to do so

Reply:
Updated to `0.6.8-pre` for now, we can undo later if you want to stick with Rust norms